lib/nylas/label.rb

```diff
@@ -17,11 +17,11 @@ class Label
 
     attribute :id, :string
     attribute :account_id, :string
-
```

Comment:
This change is made for consistency with the very similar model called [Folder](https://github.com/nylas/nylas-ruby/blob/main/lib/nylas/folder.rb#L19).